.appveyor.yml

```diff
@@ -39,7 +39,7 @@ environment:
     qtver: 6.5
     qtbin: msvc2019_arm64
     msvcname: Visual Studio 17 2022
-    cmake_args: -A ARM64 -DQT_HOST_PATH=C:/Qt/6.5/msvc2019_64 -DENABLE_SPOUT=OFF -DENABLE_QTWEBENGINE=OFF
+    cmake_args: -A ARM64 -DQT_HOST_PATH=C:/Qt/6.5/msvc2019_64 -DQT_PATHS=C:/Qt/6.5/msvc2019_arm64/bin/qtpaths.bat -DENABLE_SPOUT=OFF -DENABLE_QTWEBENGINE=OFF

```

Comment:
I guess `C:/Qt/6.5/msvc2019_arm64/bin/qtpaths.bat` should be run for ARM64 environment.

Reply:
Not, not run, it's not a batch file to set environment variables like vsvars.bat. 
I can show you what the file is if you are curious. But x-compilation is a bit complicates, that's why we need *both* Qt compiled for x64 and for ARM64 at the same machine at the same time.

It uses qmake, moc, rcc, from the HOST (x64), but when windeployqt copies files it needs to be the TARGET (ARM64) ones. qtpaths tells windeployqt which folder to use when copying the files.